frontend/control-center/src/components/Switch/index.module.scss

```diff
@@ -0,0 +1,38 @@
+.reactSwitchCheckbox {
```

Comment:
nitpick: Since every class in this file is prefixed with `reactSwitch` the prefix seems unnecessary